src/VirtualClient/VirtualClient.Actions/Prime95/Prime95Executor.cs

```diff
@@ -291,6 +291,8 @@ await this.Logger.LogMessageAsync($"{this.TypeName}.ExecuteWorkload", telemetryC
                         {
                             await this.WaitAsync(explicitTimeout, cancellationToken);
                             process.SafeKill();
+                            // await process.WaitForExitAsync(cancellationToken);
```

Comment:
Make sure this doesn't add delay to unit tests, if it does, find a way to mock it